gauge/specification.go

```diff
@@ -310,9 +310,19 @@ func (t *TearDown) Kind() TokenKind {
 }
 
 type Tags struct {
-	Values []string
+	Values [][]string
 }
 
+func (tags *Tags) Add(values []string) {
+	tags.Values = append(tags.Values, values)
+}
+
+func ToArray(values [][]string) (val []string) {
```

Comment:
`gauge.ToArray` seems to be called only for `tags.Values` for flatenning a 2d array into one. can we instead have the method on tags struct and avoid the scatter of `gauge.ToArray` everywhere?